manage.py

```diff
@@ -251,6 +250,32 @@ def demo_db():
     for i in range(5):
         add_passover(timestamp=now + timedelta(minutes=i*10))
 
+    print("Database has been seeded with demo data.")
+
+
+def import_commands():
+    """Imports commands from the CommandDocs.txt file in ex2_ground_station_software.
+    """
+    try:
+        with open("ex2_ground_station_software/CommandDocs.txt", 'r') as f:
```

Comment:
Try to stick with single quotes/keep the formatting consistent.
```suggestion
        with open('ex2_ground_station_software/CommandDocs.txt', 'r') as f:
```